<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exludech</class>
 <widget class="QDialog" name="exludech">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exclude channels</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch12</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch13</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch14</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch15</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch16</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_17">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch17</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_18">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch18</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_19">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch19</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_20">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch20</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_21">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch21</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_22">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch22</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_23">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch23</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_24">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch24</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_25">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch25</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_26">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch26</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_27">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch27</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_28">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch28</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_29">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch29</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_30">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch30</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_31">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch31</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_32">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch32</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_33">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch33</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_34">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch34</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_35">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch35</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_36">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch36</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_37">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch37</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_38">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch38</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_39">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch39</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_40">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch40</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_41">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch41</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_42">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch42</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_43">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch43</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_44">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch44</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_45">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch45</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_46">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch46</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_47">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch47</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_48">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch48</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_49">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch49</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_50">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch50</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_51">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch51</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_52">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch52</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_53">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch53</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_54">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch54</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_55">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch55</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_56">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch56</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_57">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch57</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_58">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch58</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_59">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch59</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_60">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch60</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_61">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch61</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_62">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch62</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_63">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch63</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_64">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ch64</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Check All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>exludech</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>exludech</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
